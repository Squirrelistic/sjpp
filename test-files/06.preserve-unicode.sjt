{
    "hello": "Zażółć gęślą jaźń",
    "greeting": "{{.greetingText}}"
}