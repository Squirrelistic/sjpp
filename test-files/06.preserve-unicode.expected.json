{
    "greeting": "リスは素晴らしいです",
    "hello": "Zażółć gęślą jaźń"
}